### DISCUSSION TITLE
Search in all documents at once and comparison with private gpt?

### DISCUSSION THREAD

**Discussion by maxime1992** [open-webui/open-webui#1153] [Q&A]
Hello, thanks a lot for the project!

I was wondering how it is different from private gpt when it comes to documents?

Is there a way to search in all the documents at once or just using `#` to reference a document?

Thanks!

**Comment by pwab** [comment]
See https://github.com/open-webui/open-webui/discussions/995.

Related https://github.com/open-webui/open-webui/issues/529
